setup.py

```diff
@@ -300,7 +300,8 @@ def run(self):
             raise RuntimeError("No build directory. Run `python setup.py build` before trying to run tests.")
         os.environ['GRAMPS_RESOURCES'] = '.'
         all_tests = unittest.TestLoader().discover('.', pattern='*_test.py')
-        unittest.TextTestRunner(verbosity=self.verbose).run(all_tests)
+        unittest.TextTestRunner(
+            verbosity=2 if self.verbose else 0).run(all_tests)
```

Comment:
The distutils package (on which our setup code is based) states that -v (verbose) is the default; -q (quiet) makes things quieter.
What we had before for verbose, wasn't.